code/modules/shuttle/emergency.dm

```diff
@@ -31,6 +31,11 @@
 	var/hijack_hacking = FALSE
 	var/hijack_announce = TRUE
 
+	// measures the last world time the emag event triggered
+	var/emaglast
+	// the actual time until emag again
+	var/emagcooldown = 20
```

Comment:
This can be a define as it doesn't change.
#define EMAG_COOLDOWN 2 SECONDS

Then have a section after the code to undefine it:
#undef EMAG_COOLDOWN